R/input-time-series.R

```diff
@@ -320,7 +320,7 @@ prepare_input_time_series_anc <- function(anc, shape) {
 
   ## Check if art is object or file path
   if(!inherits(anc, c("spec_tbl_df","tbl_df","tbl","data.frame" ))) {
-    anc <- readr::read_csv(anc, show_col_types = FALSE)
+    anc <- read_anc_testing(anc)
```

Comment:
Do you need the `area_name` from this one? (Does that get used in any of the time series code, etc.?)

Reply:
No `area_name` is pulled back in after merging on `area_id` with the boundaries file so not needed from the anc data:  https://github.com/mrc-ide/naomi/pull/283/files#diff-41ca6a0b90b4c5a022737ba75b3a161ccd69eaa2258feb8cb628ed2bd69733bdR30